seqr/utils/file_utils.py

```diff
@@ -35,7 +35,7 @@ def file_iter(file_path, byte_range=None):
         for line in _google_bucket_file_iter(file_path, byte_range=byte_range):
             yield line
     else:
-        with open(file_path) as f:
+        with open(file_path, 'rb') as f:
```

Comment:
`'rb'` is necessary because when the file contents unicodes the Python 2 alway read out `str` data without decoding. It will cause errors when try to encode these `str` data back to bytes.
The issue can be solved by reading with binary mode and decoding them when the data are determined to be text.